conans/client/output.py

```diff
@@ -8,12 +8,29 @@
 
 
 def colorama_initialize():
+    if "NO_COLOR" in os.environ:
+        return False
+
+    clicolor_force = get_env("CLICOLOR_FORCE")
+    if clicolor_force is not None and clicolor_force != "0":
+        import colorama
+        colorama.init(convert=False, strip=False)
+        return True
+
+    isatty = hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
+
+    clicolor = get_env("CLICOLOR")
+    if clicolor is not None:
+        if clicolor == "0" or not isatty:
+            return False
+        import colorama
+        colorama.init()
+        return True
```

Comment:
I think this should work as well
```suggestion
    if not get_env("CLICOLOR", False) or not isatty:
        return False
    if get_env("CLICOLOR", False);
        import colorama
        colorama.init()
        return True
```